openipam/network/models.py

```diff
@@ -146,7 +146,7 @@ class DhcpOption(models.Model):
     comment = models.TextField(blank=True, null=True)
 
     def __str__(self):
-        return "%s_%s" % (self.id, self.name)
+        return "%s" % self.name
```

Comment:
This was here to make the dhcp options to dhcp groups admin show the id along with the name (especially in the drop down when adding a dhcp option to group).  A lot of things document the DHCP group by id, and the names are not necessarily standardized.